consensus/polybft/state.go

```diff
@@ -840,6 +844,35 @@ func (s *State) bucketStats(bucketName []byte) *bolt.BucketStats {
 	return stats
 }
 
+// getProposerCalculatorSnapshot gets latest proposer calculator snapshot
+// we are keeping two snapshots just in case if latest write mess up snapshot
```

Comment:
I'd just remove this, since there is only one `ProposerSnapshot` in the db.
```suggestion
```

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/1018/commits/5b5dee9d1e9ef84ccabad386f0d57ec61aa962fe